script/stream-commp.js

```diff
@@ -14,13 +14,13 @@ const main = async (...args) => {
       content.byteLength
     )
 
-    const result = await CommP.build(buffer)
+    const result = await Piece.build(buffer)
 
     console.log(`
 CommPCid: ${result.link()}
-Payload:                ${result.size} bytes
+Payload:                ${result.contentSize} bytes
 Unpadded piece:         ${result.paddedSize} bytes
```

Comment:
I'm happy to change filed names to whatever makes most sense, I don't want to change labels here however because it is meant to produce same one as the one here https://github.com/filecoin-project/go-fil-commp-hashhash/tree/master/cmd/stream-commp

Reply:
ok, let's keep uniform with them...